### DISCUSSION TITLE
User feature ideas for upcoming v1.0.0 release

### DISCUSSION THREAD

**Discussion by dasupradyumna** [dasupradyumna/launch.nvim#9] [Ideas]
### IMPORTANT

Development is on-going for v1.0.0 on **release/v1-0-0** branch, with a revamped user interface for configuring tasks, debuggers and user variables. The core functionality will remain the same as **main** branch *for the most part*. The [wiki](https://github.com/dasupradyumna/launch.nvim/wiki) section has been opened documenting all the settings and options for this release.

Please feel free to provide feedback about the plugin regarding any features or changes you would like to see in the upcoming release. Based on the feasibility, these features will be added to the overall roadmap.

Thank you for using **launch.nvim**! ❤️

**Comment by Patitotective** [comment]
Hi, I was looking through this plugin and I found it looked very nice, it is simple and perfect for my use case. There is only one thing that I think it's missing, and it's to be able to define global tasks. I want to be able to configure tasks that would work on all directories, is it possible?

**Reply by dasupradyumna** [reply]
Yes, I plan to add something like that soon. This is definitely on my to-do list of features! :)

But my apologies, development on this plugin has been slow due to my hectic work schedule.
